internal/getproviders/package_authentication.go

```diff
@@ -37,16 +37,29 @@ type PackageAuthenticationResult struct {
 
 func (t *PackageAuthenticationResult) String() string {
 	if t == nil {
-		return "unauthenticated"
+		return "unsigned"
 	}
 	return []string{
 		"verified checksum",
-		"official provider",
-		"partner provider",
-		"community provider",
+		"signed by HashiCorp",
+		"signed by trusted partner",
```

Comment:
should we clarify this to "a HashiCorp-trusted partner" or similar?

Reply:
I wonder if that is too verbose, perhaps just `HashiCorp partner`?